### ISSUE TITLE
Cannot find module '../../lib/layout_creator'

### ISSUE THREAD

**Issue by terry-permeance** [liferay/generator-liferay-theme#11]
With 7.1.5 it appears that a file is missing:

```
$ yo liferay-theme:layout
module.js:341
    throw err;
    ^

Error: Cannot find module '../../lib/layout_creator'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (C:\Users\terry_000\Documents\Data\src\liferay\bootcamp\node_modules\generator-liferay-theme\generators\layout\index.js:6:21)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

**Comment by Robert-Frampton** [comment]
Hey @terry-permeance 

Can you tell me what os you're using? Along with your node and npm versions?

**Comment by Robert-Frampton** [comment]
Fixed with v7.1.6.
